xml/System.Windows.Forms/SplitterPanel.xml

```diff
@@ -359,14 +317,7 @@
       </ReturnValue>
       <Docs>
         <summary>Occurs when the value of the <see cref="P:System.Windows.Forms.SplitterPanel.Dock" /> property changes. This event is not relevant to this class.</summary>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This event is not relevant to this class. For more information about handling events, see [Handling and Raising Events](~/docs/standard/events/index.md).  
```

Comment:
I thought that this cross-reference for more information about an event that can't be handled is particularly useful.

Reply:
You think I should keep it 😉 ?